Rear Rider/Rear Rider/BlueTooth/BLEManager.swift

```diff
@@ -174,23 +164,29 @@ class BLEManager: NSObject, ObservableObject, CBCentralManagerDelegate, CBPeriph
         }
         
         print("Found \(characteristics.count) characteristics.")
-        addStatusMessage(message: "Found \(characteristics.count) characteristics.")
+        log.addLog(from: "BT", message: "Found \(characteristics.count) characteristics.")
         
         for characteristic in characteristics {
             print(characteristic.description)
             
             if characteristic.uuid.isEqual(CBUUIDs.BLECharacteristicUUID) {
                 reverseCharacteristic = characteristic
-                addStatusMessage(message: "Reverse Characteristic set")
+                log.addLog(from: "BT", message: "Reverse Characteristic set")
                 connected = true
             }
-            
             else if characteristic.uuid.isEqual(CBUUIDs.BLENotifyCharacteristicUUID) {
                 notifyCharacteristic = characteristic
-                addStatusMessage(message: "Notify Characteristic set")
+                log.addLog(from: "BT", message: "Notify Characteristic set")
+                connected = true
+                
+                //peripheral.setNotifyValue(true, for: notifyCharacteristic)
```

Comment:
```suggestion
```

Reply:
Some code I leave commented because I might need it later depending on where the app is going.

Reply:
If you ever need to revert it, the entire git history shows you the code at any commit. You can always find it that way